src/SelectedBeast.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react'
+import Modal from 'react-bootstrap/Modal';
+import  Image  from 'react-bootstrap/Image';
+// import Button from 'react-bootstrap/Button';
+
+export default class SelectedBeast extends Component {
+  render() {
+    return (
+      <div>
+        <Modal show={this.props.showModal} onHide={this.props.closeModal} >
+          <Modal.Header>
+            <Modal.Title>{this.props.title}</Modal.Title>
+          </Modal.Header>
+          <Modal.Body>
+            {this.props.description}
+          </Modal.Body>
+          <Image src={this.props.image_url} alt={this.props.title} onclick={this.handleClick} className="img-fluid"/>
```

Comment:
Props here inside `<Modal` are showing as undefined. Why might that be? Double check how props are being passed to the component. Grab a TA to get ya goin if you can 👍